test/common.py

```diff
@@ -283,7 +283,7 @@ def remove_prefix(text, prefix):
                 return text[len(prefix):]
             return text
         munged_id = remove_prefix(self.id(), "__main__.")
-        expected_file = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+        expected_file = os.path.join(os.path.dirname(os.path.realpath(__main__.__file__)),
```

Comment:
Why is this needed?

Reply:
Without it, we always place expect files in the same place as `test/common.py` We don't want this for onnx-pytorch tests, which don't live in the PyTorch test directory.